### ISSUE TITLE
Provide the definition of SD, HD and UHD terms

### ISSUE THREAD

**Issue by vladimir-kazakov** [Dash-Industry-Forum/DASH-IF-IOP#173]
DASH-IF IOP 4.1 uses terms like SD, HD, UHD, and so forth, but there's no strict definition of what these terms really mean. It seems like different guidelines, specifications and sources in the Internet put different meanings into these terms. For example, someone may treat HD as 720p only, but others like DASH-IF IOP also treat 1080p as HD. Some of them even treat 1440p as HD. As a result, the meaning of these terms is not clear.

It would be nice if DASH-IF IOP would provide the strict definition of these terms. I mean specific numbers, which would make it clear which resolutions are SD, which are HD, etc. Something like: SD is any resolution up to 719p, HD is any resolution from 720p up to 1080p, and any resolution above HD (from 1081p) is UHD. This is just an example, because I couldn't find the real meaning of these terms, but it should illustrate the expected format of the definition of the mentioned terms.

Making it clear what these terms mean would help people to better understand how to create content, to verify whether it follows some guidelines or specifications, and it would also help developers of media tools to implement certain behavior, if there's a need to do something special based on the usage of these terms.

**Comment by haudiobe** [comment]
addressed in v4.12

**Comment by vladimir-kazakov** [comment]
Do I understand correctly that this issue was addressed by adding the following sentence?

> This document uses the terms SD, HD and UHD to the best knowledge of the community, but does not provide requirements for the formats, but primarily specifies the signaling and required receiver capabilities.

If so, then I don't see how and what it solves or clarifies. I opened this issue because "the best knowledge of the community" varies, which causes problems. I was hoping that the DASH-IF IOP can provide strict definitions of these terms, so that everyone would be on the same page when using them.
